lib/defines.include.php.sample

```diff
@@ -32,6 +32,7 @@ define('CSR_DOMAIN', 'csrdelft.nl');
 
 # urls ZONDER trailing slash
 define('CSR_ROOT', 'https://' . CSR_DOMAIN);
+define('ASSETS_DIR', CSR_ROOT . '/assets');
```

Comment:
Als het goed is vragen views en templates iets in de trant van ``->getLayout()`` die de juiste layout map teruggeeft in wat nu assets heet (vroeger layout) dus is deze niet nodig, wat ook helpt omdat _PATH vs _DIR misschien verwarrend is.